### PR TITLE
Login page

### PR DESCRIPTION
**Pull request by RobertMrowiec**

### CODE REVIEW

File: app/components/login.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import Component from '@ember/component';
+
+export default Component.extend({
+    classNames: ['loginPage'],
+    email: '',
+    password: '',
+    actions: {
+        signin() {
```
Comment (Rxbsxn) [inline app/components/login.js]: What do you think about `signIn`?

File: app/components/login.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+            console.log(this.email, this.password)
```
Comment (Rxbsxn) [inline app/components/login.js]: ✂️

Comment (RobertMrowiec) [reply app/components/login.js]: Yea yea, I know :P

File: app/components/login.js (hunk 1 above)
Comment (RobertMrowiec) [reply app/components/login.js]: Sure thing, camelCase should be there

File: ember-cli-build.js [hunk 3]
```diff
@@ -3,22 +3,24 @@
...
 
 module.exports = function(defaults) {
-  let app = new EmberApp(defaults, {
-    // Add options here
-  });
+	let app = new EmberApp(defaults, {
+		sassOptions: {
+			extension: 'sass'
+		}
+	});
```
Comment (Rxbsxn) [inline ember-cli-build.js]: 4 tabs -> 2 tabs

File: app/components/login.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import Component from '@ember/component';
+
+export default Component.extend({
+	classNames: ['loginPage'],
+	email: '',
+	password: '',
```
Comment (Rxbsxn) [inline app/components/login.js]: newline here

File: app/components/navbar.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import Component from '@ember/component';
+import { inject as service } from '@ember/service'
+
+export default Component.extend({
+	auth: service(),
+	isLogged: false,
```
Comment (Rxbsxn) [inline app/components/navbar.js]: newline here

**Comment by Rxbsxn** [comment]
For using bootstrap in Ember you can use https://www.ember-bootstrap.com/#/

**Comment by Rxbsxn** [comment]
Also please take a look at https://github.com/simplabs/ember-simple-auth, it will be useful for session management

**Pull request merged**